source/mir/random/tinflex/package.d

```diff
@@ -166,7 +167,22 @@ struct Tinflex(Pdf, S)
 
     S pdf(S x) @property const
     {
-        return _pdf(x);
+        import mir.internal.math : exp;
+        bool isLog = true;
```

Comment:
`pdf` is needed for plotting and it also allows to distinguish whether a logarithmic pdf or normal pdf was provided